packages/viewer/src/components/viewer-dom-element/viewer-dom-element.tsx

```diff
@@ -183,6 +205,12 @@ export class ViewerDomElement implements HTMLDomRendererPositionableElement {
   @Prop({ reflect: true })
   public interactionsOff = false;
 
+  /**
+   * An event that is emitted any property on the dom group changes
```

Comment:
nit: missing a `when` here -> `An event that is emitted when any property on the dom group changes`